integration/raft/config_test.go

```diff
@@ -1806,3 +1807,51 @@ func updateEtcdRaftMetadata(network *nwo.Network, peer *nwo.Peer, orderer *nwo.O
 		return newMetadata
 	})
 }
+
+func mutateConsensusMetadata(originalMetadata []byte, f func(md *etcdraft.ConfigMetadata)) []byte {
+	metadata := &etcdraft.ConfigMetadata{}
+	err := proto.Unmarshal(originalMetadata, metadata)
+	Expect(err).NotTo(HaveOccurred())
+
+	f(metadata)
+
+	newMetadata, err := proto.Marshal(metadata)
+	Expect(err).NotTo(HaveOccurred())
+	return newMetadata
+}
+
+func updateOrdererMSPAndConsensusMetadata(network *nwo.Network, peer *nwo.Peer, orderer *nwo.Orderer, channel, orgID string, mutateMSP nwo.MSPMutator, f func(md *etcdraft.ConfigMetadata)) {
```

Comment:
I don't really fully understand what's going on: we update the orderer config (which we didn't do before) but we don't seem to assert that anything changed in the test - what am I missing?

Reply:
We're doing the same config updates as previously, it's just that these two are combined into a single update, which would have failed before the fix for FAB-18192 since the new consenter cert was being validated against the existing channel config instead of the in-flight (simulated) config. 

That being said, I really dislike how the existing update helpers that I awkwardly incorporated here are written since it makes following things extremely confusing to read/review as well as enhance to update multiple things at once. I wrote this originally with release-2.2 in mind (where this test has already been merged), so I avoided using the configtx library for the updates. I should probably go ahead and rewrite things here in master to utilize that. I think it'll make everything much clearer and easier to maintain.

Reply:
Understood, thanks!